README.md

```diff
@@ -42,6 +42,16 @@ contributions to the project. Feel free to open issues and send pull requests!
 
 ![Run](media/run.png)
 
+* Create run configurations from Project navigator.
```

Comment:
Let's put these setup steps into the Installation section and reformat it to so these steps fit in. I believe these steps should be mentioned as optional or maybe given in the form of tips. I feel that Features is not the best section for them, since it gives the general overview of the functions.

Reply:
Understood.  Based on the other issue, I had already decided that Features should be short and sweet and that another section was needed maybe something like Plugin Usage Details.  The document would then be ordered Features, Requirements, Installation, Usage Details, Known Issues, Source Code, License.

Reply:
@kaybee335 Agreed.

Reply:
I am going to update the README.md to make it clearer that the plugin is for uploading/running/deleting **micropython files** on the device and NOT for flashing the **micropython firmware image** onto the device.  

I am also going to add something along the lines of 'connected micropython device with recent micropython image already flashed on the device' to the Requirements section.

I think this will help allay some of the confusion that has occurred.